tron/mesos.py

```diff
@@ -230,6 +242,15 @@ def create_task(
         extra_volumes,
         serializer,
     ):
+        if not self.runner:
+            return MesosTask(
```

Comment:
When the MesosCluster starts disabled, self.runner is None, so I can't get TASK_CONFIG_INTERFACE from the runner.

Maybe I should just give up getting it from the runner? I could assume we're using the MesosTaskConfig and import it directly.

Reply:
OK, yeah, it makes more sense to not return anything instead of a dummy MesosTask. I was using the MesosTask to transition the action run, but now I'm doing that directly in action run.